src/utils/text_display.jl

```diff
@@ -27,6 +28,11 @@ end
 function human_readable_name(name::Vector{Symbol}; title_case::Bool=false)::Vector{String}
     return human_readable_name(String.(name); title_case=title_case)
 end
```

Comment:
Not related to this PR but now I'm looking at this, I don't think this function is working as expected.

Should this really be:

```julia
function human_readable_name(names::Vector{Symbol}; title_case::Bool=false)::Vector{String}
    return human_readable_name.(names; title_case=title_case)
end
```